src/main/scala/io/iohk/ethereum/domain/Transaction.scala

```diff
@@ -10,13 +10,16 @@ object Transaction {
   val GasLength = 32
   val ValueLength = 32
 
+  def apply(nonce: BigInt, gasPrice: BigInt, gasLimit: BigInt, receivingAddress: Address, value: BigInt, payload: ByteString): Transaction =
+    Transaction(nonce, gasPrice, gasLimit, Some(receivingAddress), value, payload)
+
 }
 
 case class Transaction(
   nonce: BigInt,
   gasPrice: BigInt,
   gasLimit: BigInt,
-  receivingAddress: Address,
+  receivingAddress: Option[Address],
```

Comment:
Yes, both `Address.empty` and the `ìsEmpty` function where removed from `Address.scala` in this PR as they were no longer used :+1: